ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/wan.py

```diff
@@ -63,17 +63,14 @@ def wan_listen_ranges(self: SharedUtils) -> list:
         )
 
     @cached_property
-    def cv_pathfinder_role(self: SharedUtils) -> str | None:
+    def cv_pathfinder_transit_mode(self: SharedUtils) -> str | None:
```

Comment:
I think we could simplify some other checks by returning None here also for the `"none"` case.